sormas-backend/src/main/resources/META-INF/ehcache.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+  ~  SORMAS® - Surveillance Outbreak Response Management & Analysis System
+  ~  Copyright © 2016-2022 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+  ~  This program is free software: you can redistribute it and/or modify
+  ~  it under the terms of the GNU General Public License as published by
+  ~  the Free Software Foundation, either version 3 of the License, or
+  ~  (at your option) any later version.
+  ~  This program is distributed in the hope that it will be useful,
+  ~  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  ~  GNU General Public License for more details.
+  ~  You should have received a copy of the GNU General Public License
+  ~  along with this program. If not, see <https://www.gnu.org/licenses/>.
+  ~
+  -->
+
+<ehcache>
```

Comment:
`Cachable` Annotation is added to multiple entities, e.g., `Area`. I think the list should match the newly annotated types?

Reply:
those will have a cache region without this limit, since we don't expect so many of them anyway it was not needed.
the limit itself from my point of view can be increased, but let's see at a later point after we see it in production